docs-ref-conceptual/eclipse/java-sdk-azure-get-started-eclipse.md

```diff
@@ -0,0 +1,419 @@
+---
+title: Get started with Azure for Java using Eclipse
+description: Get started with basic use of the Azure libraries for Java with your own Azure subscription.
+keywords: Azure, Java, SDK, API, authenticate, get-started
+author: roygara
+ms.author: v-rogara
+manager: timlt
+ms.date: 10/30/2017
+ms.topic: get-started-article
+ms.prod: azure
+ms.technology: azure
+ms.devlang: java
+ms.service: multiple
+---
+
+# Get started with the Azure using Eclipse
+
+This guide walks you through setting up a development environment with an Azure service principal. It also provides you with sample code that creates and uses resources in your Azure subscription using the Azure libraries for Java.
+
+## Prerequisites
+
+- An Azure account. If you don't have one, [get a free trial](https://azure.microsoft.com/free/)
+- [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/quickstart) or [Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-az-cli2).
+- [Eclipse](http://www.eclipse.org/downloads/)
```

Comment:
Point out that they should be running the latest stable Eclipse release with JDK 8. We can't have them booting up some version from 2008, our toolkits likely won't work.